# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pciutils/cross.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2007-06-20 20:47:02.000000000 +0200
+++ ./Makefile	2007-08-30 18:30:27.389589657 +0200
@@ -7,6 +7,7 @@
 VERSION=2.2.6
 DATE=2007-06-20
 
+RANLIB=ranlib
 PREFIX=/usr/local
 SBINDIR=$(PREFIX)/sbin
 SHAREDIR=$(PREFIX)/share
--- ./lib/Makefile.orig	2007-08-30 19:23:34.967239355 +0200
+++ ./lib/Makefile	2007-08-30 19:24:29.430343031 +0200
@@ -55,7 +55,7 @@
 $(PCILIB): $(OBJS)
 	rm -f $@
 	ar rcs $@ $^
-	ranlib $@
+	$(RANLIB) $@
 
 $(PCILIBPC): $(PCILIBPC).in
 	sed <$< >$@ -e 's,@PREFIX@,$(PREFIX),' \
